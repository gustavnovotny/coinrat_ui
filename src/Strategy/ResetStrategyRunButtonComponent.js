// @flow
import React from "react"
import {Button} from "@blueprintjs/core"
import {Tooltip2} from "@blueprintjs/labs"

type Props = {
  onClick: () => void,
}

const ResetStrategyRunButtonComponent = ({onClick}: Props) => {
  return <Tooltip2 content="Reset selection of strategy run and see all orders in storage.">
    <Button
      style={{marginTop: 1 + 'px'}}
      className="pt-intent-warning"
      iconName="pt-icon-filter-remove"
      onClick={onClick}
    />
  </Tooltip2>
}

export default ResetStrategyRunButtonComponent
