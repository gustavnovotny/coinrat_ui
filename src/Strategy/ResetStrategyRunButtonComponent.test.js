// @flow
import React from 'react'
import renderer from 'react-test-renderer'
import ResetStrategyRunButtonComponent from "./ResetStrategyRunButtonComponent"

it('renders correctly', () => {
  const tree = renderer.create(<ResetStrategyRunButtonComponent onClick={() => undefined}/>).toJSON()
  expect(tree).toMatchSnapshot()
})
