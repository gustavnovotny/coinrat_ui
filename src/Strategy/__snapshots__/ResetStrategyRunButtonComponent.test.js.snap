// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span>
  <div
    className="pt-popover-target"
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <button
      className="pt-button pt-icon-filter-remove pt-intent-warning"
      disabled={undefined}
      onClick={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      style={
        Object {
          "marginTop": "1px",
        }
      }
      tabIndex={0}
      type="button"
    />
  </div>
</span>
`;
